examples/sampling/graphbolt/quickstart/cora/metadata.yaml

```diff
@@ -47,3 +47,26 @@ tasks:
       name: labels
       path: nodeclassification/test-label.npy
     type_name: null
+- name: link_prediction
+  train_set:
+  - data:
+    - format: numpy
+      in_memory: true
+      name: node_pairs
+      path: linkprediction/train-node-pairs.npy
+    type_name: null
+  test_set:
```

Comment:
`validation_set` is omitted on purpose?

Reply:
Yes, same as https://github.com/dmlc/dgl/blob/master/tutorials/blitz/4_link_predict.py, this tasks is too small to have a validation set.